applications/FemToDemApplication/custom_processes/extend_pressure_condition_process.cpp

```diff
@@ -278,7 +278,9 @@ void ExtendPressureConditionProcess<2>::Execute()
     std::vector<IndexType> ToEraseConditionsId;
     this->GetMaximumConditionIdOnSubmodelPart(maximum_condition_id);
 
-    for (ModelPart::ElementsContainerType::ptr_iterator itElem = mr_model_part.Elements().ptr_begin(); itElem != mr_model_part.Elements().ptr_end(); ++itElem) {
+    for (ModelPart::ElementsContainerType::ptr_iterator itElem = mr_model_part.Elements().ptr_begin(); 
```

Comment:
Should be it_elem, and the code can be simplified with auto
```suggestion
    for (auto itElem = mr_model_part.Elements().ptr_begin(); 
```

idem for the followings

Reply:
okkey, auto abuser haha

Reply:
![imagen](https://user-images.githubusercontent.com/19991680/49234983-5d265680-f3f9-11e8-8a6c-4795c7dc8cbb.png)